cpp/src/bitmask/bit_mask.cuh

```diff
@@ -115,34 +113,31 @@ inline gdf_error put_element(bit_mask_t element, bit_mask_t *device_element) {
  *  @return GDF_SUCCESS on success, the RMM or CUDA error on error
  */
 inline gdf_error create_bit_mask(bit_mask_t **mask,
-                                 gdf_size_type number_of_records,
+                                 gdf_size_type number_of_bits,
                                  int fill_value = -1,
-                                 gdf_size_type padding_bytes = 64) {
-  //
-  //  To handle padding, we will round the number_of_records up to the next
-  //  padding boundary, then identify how many element that equates to.  Then we
-  //  can allocate the appropriate amount of storage.
+                                 gdf_size_type allocation_quantum = 64) 
```

Comment:
I think `padding_bytes` was better. `allocation_quantum` is not an intuitive name. Plus, the doc comment still says `padding_bytes`.

Reply:
Don't you think that "padding bytes" implies that there is _always_ that many bytes used in padding? (This is how I know the term "padding" to be used; and it's how it's used in CSS, for example)

Reply:
`alignment` is the term I would choose.  I think it's more common than `allocation_quantum`. :)

Reply:
But "alignment" usually (always, in my experience) regards the initial address of the data, not the amount of space allocated.

Reply:
Alignment and padding are two different things. 

>  "padding bytes" implies that there is always that many bytes used in padding?

I see what you mean. How about `padding_boundary`?

Reply:
From the arrow spec (which I believe is the origin of this): [italics are my addition]
> For performance reasons it preferred/recommended to _align buffers to a 64-byte boundary_ and _pad to a multiple of 64 bytes_, but this is not absolutely necessary. The rationale is discussed in more details below.

So if we were talking about `alignment`, "boundary" might be the right term. But I think in this case we're talking about a "padding factor". Multiples are products of factors. "pad to a multiple of 64 bytes", therefore 64 is the "padding factor". 

So I suggest the name `padding_factor`.

Reply:
@harrism : I'm fine with that, since while its meaning is not obvious, it doesn't have an obvious _incorrect_ interpretation...

Reply:
We seem to have about a 36 hour latency between comments and commits. Any chance you can make this one soon?